config/main.py

```diff
@@ -1828,6 +1841,113 @@ def delete_snmptrap_server(ctx, ver):
+    if vlan_id_is_valid(vid2) is False:
+        ctx.fail("{} is not within allowed range of 1 through 4094".format(vlan2))
+
+    if vid2 <= vid1:
+        ctx.fail(" vid2 should be greater than vid1")
+
+#
+# Return a string with ranges separated by hyphen.
+#
+def get_hyphenated_string(vlan_list):
+    vlan_list.sort()
+    G = (list(x) for _,x in groupby(vlan_list, lambda x,c=count(): next(c)-x))
+    hyphenated_string = ",".join("-".join(map(str,(g[0],g[-1])[:len(g)])) for g in G)
+    return hyphenated_string
+
+#
+# 'mac' group ('config mac ...')
+#
+@config.group()
+@click.pass_context
+@click.option('-s', '--redis-unix-socket-path', help='unix socket path for redis connection')
+def mac(ctx, redis_unix_socket_path):
+    """Mac-related configuration tasks"""
+    kwargs = {}
+    if redis_unix_socket_path:
+        kwargs['unix_socket_path'] = redis_unix_socket_path
+    config_db = ConfigDBConnector(**kwargs)
+    config_db.connect(wait_for_init=False)
+    ctx.obj = {'db': config_db}
+
+@mac.command('aging_time')
+@click.argument('interval', metavar='<aging interval in sec [0-1000000], 0 to disable aging>', required=True, type=int)
+@click.pass_context
+def set_aging_time(ctx, interval):
+    db = ctx.obj['db']
+    if interval not in range(0,1000001):
+        ctx.fail("Aging timer must be in range [0-1000000]")
+    db.set_entry('SWITCH', 'switch', {'fdb_aging_time': interval})
+
+@mac.command('add')
+@click.argument('mac', metavar='<mac-address as xx:xx:xx:xx:xx:xx>', required=True)
+@click.argument('vlan', metavar='<vlan>', required=True, type=int)
+@click.argument('interface_name', metavar='<interface name>', required=True)
+@click.pass_context
+def add_mac(ctx, mac, vlan, interface_name):
+    db = ctx.obj['db']
+
+    mac_valid = bool(re.match('^' + '[\:\-]'.join(['([0-9a-f]{2})']*6) + '$', mac.lower()))
+    if mac_valid == False:
+        ctx.fail("Incorrect mac-address format!!")
+
+    mac_valid = not bool(re.match('^' + '[\:\-]'.join(['([00]{2})']*6) + '$', mac.lower()))
+    if mac_valid == False:
+        ctx.fail("Invalid (Zero) mac-address!!")
+
+    mac_valid = not bool(re.match('^' + '[\:\-]'.join(['([ff]{2})']*6) + '$', mac.lower()))
+    if mac_valid == False:
+        ctx.fail("Invalid (Bcast) mac-address!!")
+
+    mac_is_multicast = int(mac[:2]) & 1;
+    if mac_is_multicast == True:
+        ctx.fail("Invalid (Multicast) mac-address!!")
+
+    vlan_valid = bool(vlan_id_is_valid(vlan))
+    if vlan_valid == False:
+        ctx.fail("Invalid VlanId!!")
+
+    vlan_name = 'Vlan{}'.format(vlan)
+
+    if get_interface_naming_mode() == "alias":
+        interface_name = interface_alias_to_name(interface_name)
+        if interface_name is None:
+            ctx.fail("'interface_name' is None!")
+
+    if interface_name_is_valid(interface_name) is False:
+        ctx.fail("Interface name is invalid!!")
+
+    db.set_entry('FDB', (vlan_name, mac), {'port': interface_name })
```

Comment:
Currently, there is no config db support for FDB and aging time. For this, we need to to have this [PR](https://github.com/Azure/sonic-swss/pull/885), correct?